chainstate/src/detail/tests/test_framework.rs

```diff
@@ -97,40 +99,123 @@ impl BlockTestFramework {
 
         Block::new(
             vec![Transaction::new(0, inputs, outputs, 0).expect(ERR_CREATE_TX_FAIL)],
-            Some(prev_block_hash),
+            prev_block_hash,
             BlockTimestamp::from_duration_since_epoch(time::get()),
             ConsensusData::None,
         )
         .expect(ERR_CREATE_BLOCK_FAIL)
     }
 
-    pub fn genesis(&self) -> &Block {
+    pub(in crate::detail::tests) fn genesis(&self) -> &Genesis {
```

Comment:
Same here, but I suppose there is a reason behind this. 🙃

Reply:
I don't mind this being just `pub`